src/opm/parser/eclipse/EclipseState/Tables/Tables.cpp

```diff
@@ -92,14 +92,14 @@ PvtgTable::PvtgTable( const DeckKeyword& keyword, size_t tableIdx ) :
     }
 
 PvtoTable::PvtoTable( const DeckKeyword& keyword, size_t tableIdx) :
-    PvtxTable("P") {
+    PvtxTable("RS") {
```

Comment:
Basically, this means `m_outerColumnSchema` from `PvtxTable` is never used except it is used to initialize `m_outerColumn`. Although it has the wrong `Schema`, it does not looks like it causes any problem or it is used to identify anything.